hazelcast/src/main/java/com/hazelcast/client/impl/proxy/ClientClusterProxy.java

```diff
@@ -127,6 +131,11 @@ public void changeClusterVersion(@Nonnull Version version, @Nonnull TransactionO
         throw new UnsupportedOperationException();
     }
 
+    @Override
+    public boolean isEnterprise() {
```

Comment:
```suggestion
    public boolean isConnectedToEnterpriseCluster() {
```
why do you use a different method names in different places? I would rather have them the same. what do you think?

Reply:
Used different names because Cluster interface keeps cluster information. So isEnterprise means whether the cluster we are connected to is Enterprise or not.

however, in the connection manager isEnterprise would be ambigous in my opinion. Do we mean the local Hazelcast Instance or the cluster etc.